exporters/otlp/internal/transform/metric.go

```diff
@@ -289,11 +289,74 @@ func Record(r export.Record) (*metricpb.Metric, error) {
 		}
 		return gauge(r, value, time.Time{}, tm)
 
+	case aggregation.ExactKind:
+		e, ok := agg.(aggregation.Points)
+		if !ok {
+			return nil, fmt.Errorf("%w: %T", ErrIncompatibleAgg, agg)
+		}
+		pts, err := e.Points()
+		if err != nil {
+			return nil, err
+		}
+
+		return gaugeArray(r, pts)
+
 	default:
 		return nil, fmt.Errorf("%w: %T", ErrUnimplementedAgg, agg)
 	}
 }
 
+func gaugeArray(record export.Record, points []otel.Number) (*metricpb.Metric, error) {
```

Comment:
Any reason to not just call this function `array`?

Reply:
yeah I used `array` initially and the linter is not happy. There is a pacakge `go.opentelemetry.io/otel/sdk/metric/aggregator/array`.